pkg/apis/common/common.go

```diff
@@ -156,6 +156,20 @@ type SASLConfig struct {
 	PasswordSecret *corev1.SecretKeySelector `json:"passwordSecret,omitempty" protobuf:"bytes,3,opt,name=password"`
 }
 
+// SchemaRegistryConfig refers to configuration for a client
+type SchemaRegistryConfig struct {
+	// Schema Registry URL.
+	URL string `json:"url" protobuf:"bytes,1,opt,name=url"`
+	// +optional
+	// Api key is the authentication for registry
+	UserSecret *corev1.SecretKeySelector `json:"userSecret,omitempty" protobuf:"bytes,2,opt,name=user"`
+	// +optional
+	// Api secret is the authentication for registry
+	PasswordSecret *corev1.SecretKeySelector `json:"passwordSecret,omitempty" protobuf:"bytes,3,opt,name=password"`
+	// Schema ID
+	SchemaId int32 `json:"schemaId,omitempty" protobuf:"varint,4,opt,name=schemaId"`
```

Comment:
Nit: Use `SchemaID` in golang (keep `schemaId` in the `json`).

Is this field required when the registry config is in-use?

Reply:
Ok, sorry.
Yes schemaID is mandatory when config is defined.